Source/Core/DolphinQt/Config/Mapping/HotkeyControllerProfile.cpp

```diff
@@ -16,10 +16,27 @@ HotkeyControllerProfile::HotkeyControllerProfile(MappingWindow* window) : Mappin
 
 void HotkeyControllerProfile::CreateMainLayout()
 {
-  m_main_layout = new QHBoxLayout();
-
-  m_main_layout->addWidget(CreateGroupBox(
-      tr("Controller Profile"), HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE)));
+  QGridLayout* m_main_layout = new QGridLayout;
+
+  m_main_layout->addWidget(
+      CreateGroupBox(tr("Wii Remote %1").arg(1),
```

Comment:
At this point using .arg() is superfluous, we should just put the number directly in the string since it's a compile-time constant. This would also allow translators to handle the numbers appropriately for their target language.

Reply:
I can see the advantage of using the same string so it only has to be translated once.
I would be fine with either solution.